apps/ledger-live-mobile/ios/ledgerlivemobile/AppDelegate.m

```diff
@@ -183,4 +183,14 @@ - (BOOL)application:(UIApplication *)application
   return [RCTLinkingManager application:application openURL:url options:options];
 }
 
+- (BOOL)application:(UIApplication *)application
+continueUserActivity:(NSUserActivity *)userActivity
+ restorationHandler:(void (^)(NSArray<id<UIUserActivityRestoring>> *restorableObjects))restorationHandler
+{
+  NSURL *url = userActivity.webpageURL;
+  return [RCTLinkingManager application:application
+                   continueUserActivity:userActivity
+                     restorationHandler:restorationHandler];
+}
+
```

Comment:
Implement support for iOS universal links by implementing `application:continueUserActivity:restorationHandler` in `AppDelegate.m` and forwarding to React Native Linking library (`RCTLinkingManager`)